openshift/cronJob-template.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: init-job-cronjob
+  annotations:
+    description: This is Thoth Core - Init Job
+    openshift.io/display-name: 'Thoth: Init Job'
+    version: 0.1.0
+    tags: poc,thoth,ai-stacks,init-job
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to run recommendation logic of Thoth to OpenShift.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+    thoth-station.ninja/template-version: 0.1.0
+  labels:
+    app: thoth
+    template: init-job-cronjob
+    component: init-job
+
+parameters:
+  - name: THOTH_LOG_INIT_JOB
+    required: false
+    description: Log init-job actions
+    displayName: Log init-job actions
+    value: INFO
+  - description: Registry for the ImageStream to be used, lives in.
+    displayName: ImageStream Registry
+    required: true
+    name: IMAGE_STREAM_REGISTRY
+    value: "docker-registry.default.svc:5000"
+  - description: The Project where the ImageStream lives in.
+    displayName: ImageStream Project
+    required: true
+    name: IMAGE_STREAM_NAMESPACE
+    value: "thoth-infra-stage"
+  - description: Tag of the ImageStream to be used.
+    displayName: ImageStream Tag
+    required: true
+    name: IMAGE_STREAM_TAG
+    value: "latest"
+  - description: Service account to be used.
+    displayName: Service account name.
+    required: true
+    name: SERVICE_ACCOUNT_NAME
+    value: "init-job"
+
+objects:
+  - apiVersion: batch/v1beta1
+    kind: CronJob
+    metadata:
+      annotations:
+        thoth-station.ninja/template-version: 0.1.0
+      name: init-job
+      labels:
+        app: thoth
+        component: init-job
+    spec:
+      schedule: "0 */12 * * *"
+      jobTemplate:
+        spec:
+          template:
+            metadata:
+              labels:
+                app: thoth
+                component: init-job
+            spec:
+              restartPolicy: Never
```

Comment:
Change [from](https://github.com/thoth-station/init-job/blob/master/openshift/job-template.yaml#L37) to `Never`